### DISCUSSION TITLE
Need manual set `::before { content: '' }` for using before  pseudo-element?

### DISCUSSION THREAD

**Discussion by aliuq** [unocss/unocss#414] [Q&A]
<img width="750" alt="image" src="https://user-images.githubusercontent.com/11707934/148245912-01f14649-e770-4bba-a328-48e0d1df4f89.png">

**Comment by chu121su12** [comment]
As for now, see
https://github.com/antfu/unocss/pull/185#issuecomment-983815201
and 
https://github.com/antfu/unocss/issues/124#issuecomment-981532905

**Reply by aliuq** [reply]
Thank you very much
